apps/explorer/.env.example

```diff
@@ -1,3 +1,3 @@
 VITE_BACKEND_URL=
 VITE_NODE_URL=
-VITE_NETWORK_TYPE=
+VITE_NETWORK_TYPE=testnet
```

Comment:
It's hard for a developer to know what to input there. Prefilling it with the string `testnet` helps, I believe.